_rules/html-has-lang-b5c3f8.md

```diff
@@ -25,24 +25,21 @@ The root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s)
 
```

Comment:
> Note: Documents embedded into other documents, such as through iframe or object elements are not applicable because they are not web pages according to the definition in WCAG.

I think this leaves a gap in the rules because iframes are not checked by https://act-rules.github.io/rules/de46e4 (language of parts) or https://act-rules.github.io/rules/b5c3f8 (language of page). `lang` is not inherited from the parent document according to this comment https://stackoverflow.com/a/53759156

The seems right because iframe security restrictions (e.g. sandbox) are supposed to prevent the iframe documents from accessing parent document state (including lang)

Reply:
That's a good point. Updating.

Reply:
Will move this update to a separate PR.